src_Core/RISCY_OOO/procs/RV64G_OOO/FetchStage.bsv

```diff
@@ -816,7 +816,7 @@ module mkFetchStage(FetchStage);
    Vector#(SupSize, DecodeResult) decodeResults = ?;
    for (Integer i = 0; i < valueOf(SupSize); i = i + 1) begin
       CapMem pc = decompressPc(validValue(decodeIn[i]).pc);
-      decodeResults[i] = decode(validValue(decodeIn[i]).inst, !getUnlegalisedIntMode(pc)); // Decode 32b inst, or 32b expansion of 16b inst. Mode must be legal if we've fetched
+      decodeResults[i] = decode(validValue(decodeIn[i]).inst, !getUnlegalisedIntMode(pc));
```

Comment:
Did you mean to remove the first part of the comment?

Reply:
Nice catch :)